src/app/shared/models/contact.model.ts

```
@@ -0,0 +1,8 @@
+export interface Contact {
+  address: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)